test/topology_experimental_raft/test_mv_tablets.py

```diff
@@ -51,3 +51,25 @@ async def test_tablet_mv_simple(manager: ManagerClient):
     # We used SYNCHRONOUS_UPDATES=TRUE, so the view should be updated:
     assert [(3,2)] == list(await cql.run_async("SELECT * FROM test.tv WHERE c=3"))
     await cql.run_async("DROP KEYSPACE test")
+
+@pytest.mark.asyncio
+async def test_tablet_mv_simple_6node(manager: ManagerClient):
+    """A simple reproducer for a bug of forgetting that the view table has a
+       different tablet mapping from the base: Using the wrong tablet mapping
+       for the base table or view table can cause us to send a view update
+       to the wrong view replica - or not send a view update at all. A row
+       that we write on the base table will not be readable in the view.
+       We start a large-enough cluster (6 nodes) to increase the probability
+       that if the mapping is different for the one row we write, and the test
+       will fail if the bug exists.
+       Reproduces #16227.
+    """
+    servers = [await manager.server_add() for i in range(6)]
```

Comment:
you can use concurrent bootstrap now, it'll be a tad faster. see this pull request for details, add_servers function in particular https://github.com/scylladb/scylladb/pull/16102/files

Reply:
I'll check it out, thanks. I just copied this snippet from other tests :-)

Reply:
@kbr-scylla when is the patch with parallel bootstrap going to end up in the master?
@nyh perhaps it's only in next?